altsrc/flag.go

```diff
@@ -124,13 +124,14 @@ func (f *BoolFlag) ApplyInputSourceValue(cCtx *cli.Context, isc InputSourceConte
 	if f.set != nil && !cCtx.IsSet(f.Name) && !isEnvVarSet(f.EnvVars) && isc.isSet(f.BoolFlag.Name) {
 		value, err := isc.Bool(f.BoolFlag.Name)
 		if err != nil {
+			fmt.Println(err)
```

Comment:
If printing is required here?

Reply:
Ah nice catch. Those are debug logs. I've removed them.

Reply:
Seems to have the same problem at 
[*StringFlag.ApplyInputSourceValue](https://github.com/urfave/cli/blob/b4df361387226948b6cbaf53da539e26c7562cce/altsrc/flag.go#L145)